exercises/leap/leap.t6

```diff
@@ -28,42 +28,50 @@ for $c-data<cases>.values {
 =begin code
 {
   "exercise": "leap",
-  "version": "1.4.0",
+  "version": "1.5.1",
   "cases": [
     {
-      "description": "year not divisible by 4: common year",
+      "description": "year not divisible by 4 in common year",
       "property": "leapYear",
       "input": {
         "year": 2015
       },
       "expected": false
     },
     {
-      "description": "year divisible by 4, not divisible by 100: leap year",
+      "description": "year divisible by 2, not divisible by 4 in common year",
+      "property": "leapYear",
+      "input": {
+        "year": 1970
+      },
+      "expected": false
+    },
+    {
+      "description": "year divisible by 4, not divisible by 100 in leap year",
       "property": "leapYear",
       "input": {
         "year": 1996
       },
       "expected": true
     },
     {
-      "description": "year divisible by 100, not divisible by 400: common year",
+      "description": "year divisible by 100, not divisible by 400 in common year",
```

Comment:
"In" or "is"?  I believe these 'in' replacements should be 'is'.

Reply:
The canonical data is copied over from https://github.com/exercism/problem-specifications by the generator, so it may be worth raising the issue there.

Reply:
Introduced in problem-specifications#1468 specifically.

Reply:
Opened PR: https://github.com/exercism/problem-specifications/pull/1591

Reply:
The canonical data change is on hold. I'd like to merge this as is and then update once this has been updated upstream.

Reply:
Sounds good, bringing it in.